packages/neon-core/src/sc/ContractParam.ts

```diff
@@ -81,144 +92,235 @@ export class ContractParam implements NeonObject<ContractParamLike> {
         `hash160 expected a 40 character string but got ${value.length} chars instead.`
       );
     }
-    return new ContractParam({ type: ContractParamType.Hash160, value });
+    return new ContractParam({
+      type: ContractParamType.Hash160,
+      value: HexString.fromHex(value),
+    });
   }
 
   /**
-   * Creates an Integer ContractParam. This is converted into an BigInteger in NeoVM.
+   * Creates an Integer ContractParam. This is converted into an BigInteger in NeoVM. Value field will be a string.
    * @param value - A value that can be parsed to an BigInteger. Numbers or numeric strings are accepted.

```

Comment:
```suggestion
   * Creates an Integer ContractParam. This is converted into a BigInteger in NeoVM. Value field will be a string.
   * @param value - A value that can be parsed to a BigInteger. Numbers or numeric strings are accepted.

```
just a quick grammar thing